apps/server/src/modules/oauth-provider/controller/dto/request/reject.body.ts

```diff
@@ -0,0 +1,51 @@
+import { IsNumber, IsOptional, IsString } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class RejectBody {
```

Comment:
please call this something like OAuthRejectBody, to make clear that this is only used in this context.

Reply:
maybe even "...RejectableBody" to infer that the final (inheriting) response might actually be a successfull one